colossalai/nn/layer/moe/checkpoint.py

```diff
@@ -7,7 +8,7 @@
 from torch.optim import Optimizer
 
 from colossalai.checkpoint_io import CheckpointIO
-from colossalai.tensor.moe_tensor.api import get_ep_group
+from colossalai.tensor.moe_tensor.api import get_dp_rank, get_ep_group, get_ep_rank, get_ep_size, is_moe_tensor
 
 
 class MoeCheckpintIO(CheckpointIO):
```

Comment:
Maybe you should use `GeneralCheckpointIO`.

Reply:
done